pkgs/thycotic/dsv-cli/registry.yaml

```diff
@@ -0,0 +1,31 @@
+packages:
+  - type: github_release
+    repo_owner: thycotic
+    repo_name: dsv-cli
+    asset: dsv-{{.OS}}-{{.Arch}}
+    format: raw
+    replacements:
+      amd64: x64
+      arm64: x64
+      386: x86
+      arm64n: arm64
+      windows: win
+    supported_envs:
+      - darwin
+      - linux
+      - windows
+    checksum:
+      type: github_release
+      asset: '{{.Asset}}-sha256.txt' # Fixed
+      file_format: regexp
+      algorithm: sha256
+      pattern:
+        checksum: "^(\\b[A-Fa-f0-9]{64}\\b)"
+        file: "^\\b[A-Fa-f0-9]{64}\\b\\s+\\./(\\S+)$" # Fixed
+    link: https://docs.delinea.com/dsv/current/cli-ref/syntax.md
+    description: Delinea DevOps Secrets Vault is a secret vault automation tool for the management of credentials for applications, databases, CI/CD tools, and services.
```

Comment:
```suggestion
    description: Delinea DevOps Secrets Vault is a secret vault automation tool for the management of credentials for applications, databases, CI/CD tools, and services
```

[Remove characters `.!` from the end of the description](https://aquaproj.github.io/docs/reference/registry-style-guide#remove-characters--from-the-end-of-the-description)